x-pack/solutions/observability/plugins/observability_onboarding/public/application/quickstart_flows/otel_logs/multi_integration_install_banner.tsx

```diff
@@ -22,7 +22,7 @@ export function MultiIntegrationInstallBanner() {
 
   const { performRequest, requestState } = useInstallIntegrations({
     onIntegrationCreationFailure,
-    packages: ['system', 'kubernetes'],
+    packages: ['system_otel', 'kubernetes'],
```

Comment:
```suggestion
    packages: ['system_otel', 'kubernetes_otel'],
```

As correctly pointed out by @ChrsMark in https://github.com/elastic/hosted-otel-collector/pull/897#issuecomment-3053544374, we should also replace the `kubernetes` with `kubernetes_otel`